doc/configuration.rst

```diff
@@ -160,6 +160,9 @@ Currently available are:
 ``digipower``
   Controls a DigiPower PDU via a simple HTTP API.
 
+``digitalloggers_http``
+  Control a Digital Loggers PDU that use the legacy HTTP API.
```

Comment:
This should mention the expected format of the `host` parameter. It looks like it should be `http://mypdu`?